### ISSUE TITLE
Search functionality

### ISSUE THREAD

**Issue by ftomassetti** [lu-zero/plaid#8]
The ability to search over all the fields of patches

**Comment by ftomassetti** [comment]
For this one we have a patch pending

**Comment by lu-zero** [comment]
Your patch is filtering using python and doesn't paginate (major sin).

So far it searches by `project` and by `name`.

**Comment by ftomassetti** [comment]
Yes, it is filtering in python also because text search in SQL depends on the different engine implementations. I am not sure how the ORM deals with that. Given the size of data involved I think it is reasonable and we have an implementation ~1.5 years after the issue was opened but I will not prevent someone else to come up with a better solution :)

**Comment by lu-zero** [comment]
The solution now is in the tree ;)

**Comment by ftomassetti** [comment]
Great!

**Comment by ftomassetti** [comment]
Should we close this one? I will take a look to learn something :)

**Comment by lu-zero** [comment]
I'll close it tomorrow once I land the rest retouches.

**Comment by lu-zero** [comment]
the whoosh integration with sqlalchemy I was using is too brittle.

**Comment by lu-zero** [comment]
Found a working version but this https://github.com/dhamaniasad/Flask-WhooshAlchemy/issues/4 is missing.

**Comment by ftomassetti** [comment]
Should we close this one?
